pkg/version/version.go

```diff
@@ -42,7 +42,7 @@ var (
 	TimescaleVersionRangeString = struct {
 		Safe, Warn string
 	}{
-		Safe: ">=1.7.3 <2.0.0",
+		Safe: ">=1.7.3 <1.7.99",
```

Comment:
it feels a bit weird conditioning against a version that doesn't actually exist. Maybe we can use `<2.0.0-alpha` instead? That way we can be compatible with any theoretical future point releases.